package.json

```diff
@@ -90,9 +91,10 @@
     "json-loader": "^0.5.7",
     "node-sass": "^4.12.0",
     "node-ssi": "^0.3.2",
-    "npm": "^6.14.8",
+    "npm": "^5.6.0",
```

Comment:
why the npm downgrade?

Reply:
Updated to the latest in the recent commit.